djstripe/models/base.py

```diff
@@ -273,11 +273,21 @@ def _find_owner_account(cls, data, api_key=djstripe_settings.STRIPE_SECRET_KEY):
         """
         from .account import Account
 
-        if data and data.get("account"):
-            # try to fetch by stripe_account. Also takes care of Stripe Connected Accounts
-            stripe_account = cls._id_from_data(data.get("account"))
-            if stripe_account:
-                return Account._get_or_retrieve(id=stripe_account)
+        # try to fetch by stripe_account. Also takes care of Stripe Connected Accounts
+        if data:
+            # case of Webhook Event Trigger
```

Comment:
? WebhookEventTrigger is not a StripeModel. Am I misunderstanding this comment?

Reply:
No It's not. It is to be able to populate the `stripe_trigger_account` model field on the `WebhookEventTrigger` Account correctly [here](https://github.com/dj-stripe/dj-stripe/blob/master/djstripe/models/webhooks.py#L112)

Reply:
Okay, I see.